lib/core/test/unit/Cardano/Pool/DB/Arbitrary.hs

```diff
@@ -98,6 +107,14 @@ instance Arbitrary PoolId where
         bytes <- vectorOf 32 (elements ['a'..'z'])
         return $ PoolId $ B8.pack bytes
 
+instance Arbitrary StakePoolOwnersFixture where
+    arbitrary = StakePoolOwnersFixture <$> fmap PoolId bs <*> listOf entry
+      where
+        entry = (,) <$> fmap PoolId bs <*> fmap PoolOwner bs
+        bs = B8.replicate 32 <$> elements ['a'..'e']
```

Comment:
**[suggestion]**

rename `bs` to `genBs` or `genBytes` 

**Why?**

At first glance, a reader may interpret `bs` as a byte string. Yet, it is indeed a generator here and therefore leads to some confusion. Prefixing with `gen`  better reflect what the let binding captures.

Reply:
OK fixed.